CHANGELOG.md

```diff
@@ -1,3 +1,123 @@
+# 77.0.0
+
+In this release we made improvements to BottomNavigationBar and landed stateful ripple in the ripple (beta) component.
+
+## Breaking changes
+
+### BottomNavigationBar
+
+Removed a deprecated API:
+```
+@property(nonatomic, assign) UIEdgeInsets itemsContentInsets 
+```
+
+## New features
+
+### BottomNavigationBar
+
+#### Blur
+
+BottomNavigation can use blur in its background when not fully opaque
+```
+self.bottomNavBar.backgroundBlurEnabled = YES;
+```
+
+#### Long item titles
+
+We automatically truncate titles that go out of bounds but added a bool to allow some clients to still overflow.
```

Comment:
Tip: Nit: "...but added a *flag* to allow..."  If you're going to call-out the property's type, please surround it in \`backticks\` to get monospace font styling.

Reply:
Done.